lib/query.js

```diff
@@ -105,7 +106,8 @@ function geoservicesPostQuery (data, queriedData, params) {
     params.spatialReference = params.outSR
     params.attributeSample = data.features[0] && data.features[0].properties
     params.geometryType = getGeometryTypeFromGeojson(data)
-    return renderFeatures(queriedData, params)
+    const rFeatures = renderFeatures(queriedData, params)
```

Comment:
No reason to assign this to a variable when you just `return` it on the next line.